wagtail/tests/test_tests.py

```diff
@@ -172,6 +172,16 @@ def test_assert_can_create(self):
             },
         )
 
+        self.assertCanCreate(
+            self.root,
+            SimplePage,
+            {
+                "title": "Simple Lorem Page",
+                "content": "Lorem ipsum dolor sit amet"
+            },
+            publish=False
```

Comment:
However, here you probably want to check that the exception is raised - calling this code is probably just raising an error which will normally fail 

You probably want to do a seperate test here that checks the error you want is actually raised. 

See an example in `wagtail/tests/test_page_assertions.py`

```
    def test_is_previewable_for_broken_mode(self):
        with self.assertRaises(self.failureException):
            self.assertPageIsPreviewable(self.page, mode="broken")
```